crates/holochain/src/core/ribosome/host_fn/must_get_agent_activity.rs

```diff
@@ -56,12 +56,7 @@ pub fn must_get_agent_activity(
                 use MustGetAgentActivityResponse::*;
 
                 let result: Result<_, RuntimeError> = match (result, &call_context.host_context) {
-                    (Activity(activity), _) => Ok(activity),
-                    (Warrants(_), _) => {
-                        // TODO: we probably want to handle this differently
-                        tracing::warn!("must_get_agent_activity encountered Warrants. Returning empty activity.");
-                        Ok(vec![])
-                    }
+                    (Activity {activity, warrants: _}, _) => Ok(activity),
```

Comment:
Does `warrants` need to be bound if it isn't used? Just `..`?

Reply:
Good point, it should be used!